openedx/core/djangoapps/schedules/admin.py

```diff
@@ -9,22 +13,87 @@ class ScheduleExperienceAdminInline(admin.StackedInline):
     model = models.ScheduleExperience
 
 
+def _set_experience(db, human, modeladmin, request, queryset):
```

Comment:
I eventually understood what `db` and `human` meant - but only after reading what `ScheduleExperience.EXPERIENCES` was defined as.   Any possibility of better variable names for these 2?

Reply:
fixed